app-text/arianna/arianna-9999.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="forceoptional"
+KDE_ORG_CATEGORY="graphics"
+PVCUT=$(ver_cut 1-3)
+KFMIN=5.240.0
+QTMIN=6.5.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="EPub Reader for mobile devices"
+HOMEPAGE="https://apps.kde.org/arianna/"
+
+LICENSE="|| ( GPL-2 GPL-3 ) || ( LGPL-2.1 LGPL-3 ) || ( MIT GPL-3 ) BSD CC0-1.0 CC-BY-SA-4.0 GPL-3 GPL-3+ LGPL-2+ LGPL-2.1+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	dev-libs/kirigami-addons:6
+	>=dev-qt/qtbase-${QTMIN}:6[gui,widgets,sql,xml]
+	>=dev-qt/qthttpserver-${QTMIN}:6
```

Comment:
Erk, it requires this but it isn't packaged for Gentoo yet (i have it in my personal overlay.) How do I submit this or is it hiding somewhere already where I didn't find it?

Reply:
New package request in Gentoo Bugzilla.

Reply:
Thanks, filed as https://bugs.gentoo.org/920189